src/map/npc.cpp

```diff
@@ -5181,6 +5181,12 @@ static const char* npc_parse_mob(char* w1, char* w2, char* w3, char* w4, const c
 		return strchr(start,'\n');// skip and continue
 	}
 
+	if (delay != 5000 && (delay < 1000 || delay > UINT_MAX)) {
```

Comment:
delay is an int32 so it can never be > UINT32_MAX.
Maybe just make it delay < battle_config.mob_respawn_time.
Also to check if delay was given correctly you need to introduce w4count, like we have w1count.
Otherwise it allows setting smaller delays even if you set the battle_config to 10 seconds for example.